var/spack/repos/builtin/packages/py-alphafold/package.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import glob
+
+
+class PyAlphafold(PythonPackage, CudaPackage):
+    """AlphaFold is an AI system developed by DeepMind that predicts a
+    protein's 3D structure from its amino acid sequence. It regularly
+    achieves accuracy competitive with experiment."""
+
+    homepage = 'https://github.com/deepmind/alphafold'
+    url = 'https://github.com/deepmind/alphafold/archive/refs/tags/v2.1.1.tar.gz'
+
+    version('2.1.1', sha256='1adb6e213ba9ac321fc1acb1c563ba9b4fc054c1cebe1191bc0e2aaa671dadf7')
+
+    depends_on('python@3.7:', type=('build', 'run'))
```

Comment:
Where do you see that Python 3.7+ is required?

Also missing build dep on setuptools

Reply:
I used the dockerfile and requirements.txt as "hints". Fixed.

Reply:
The `setup.py` lists 3.6 as a supported version, and doesn't restrict the version of any dependencies. I personally wouldn't enforce any minimum version for these deps, but if you want to maintain this package I'll leave the decision to you.